ERCs/eip-5732/contracts/CommitToMintImpl.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Author: Zainan Victor Zhou <zzn-ercref@zzn.im>
+// Visit our open source repo: http://zzn.li/ercref
+
+pragma solidity ^0.8.9;
```

Comment:
Could probably do `0.8.17` unless this is needed for compatibility reasons with OZ.

Reply:
Good point